components/openai/actions/create-fine-tuning-job/create-fine-tuning-job.mjs

```diff
@@ -0,0 +1,57 @@
+import openai from "../../openai.app.mjs";
+
+export default {
+  key: "openai-create-fine-tuning-job",
+  name: "Create Fine Tuning Job",
+  description: "Creates a job that fine-tunes a specified model from a given dataset. [See the documentation](https://platform.openai.com/docs/api-reference/fine-tuning/create)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    openai,
+    model: {
+      propDefinition: [
+        openai,
+        "fineTuningModel",
+      ],
+    },
+    trainingFile: {
+      propDefinition: [
+        openai,
+        "trainingFile",
+      ],
+    },
+    hyperParameters: {
+      type: "object",
+      label: "Hyperparameters",
+      description: "The hyperparameters used for the fine-tuning job. [See details on the documentation.](https://platform.openai.com/docs/api-reference/fine-tuning/create#fine-tuning-create-hyperparameters)",
```

Comment:
"See details on" -> "See details in" and move the period at the end outside of the link.